client/src/pages/tasks/Show.js

```diff
@@ -166,6 +174,7 @@ const TaskShow = ({ pageDispatchers }) => {
   const ShortNameField = DictionaryField(Field)
   const LongNameField = DictionaryField(Field)
   const TaskParentTask = DictionaryField(Field)
+  const TaskChildrenTasks = DictionaryField(Field)
```

Comment:
Remove this line, childrenTasks does not match the dictProps of DictionaryField (it's not an input field).